template/base/neuron.py

```diff
@@ -107,6 +129,7 @@ def __init__(self, config=None):
         mandatory_config['uid'] = self.uid
         mandatory_config['ip'] = re.search(r"/ipv4/([^:]*)", self.metagraph.addresses[self.uid]).group(1)
         mandatory_config['hotkey'] = self.wallet.hotkey.ss58_address
+        mandatory_config['coldkey'] = self.metagraph.coldkeys[self.uid]
```

Comment:
this will not work, this will prompt for password and hold execution

Reply:
It doesn't require password. It has been already tested.